frontend/app.vue

```diff
@@ -17,6 +17,11 @@ const items = ref<NavigationMenuItem[]>([
     icon: 'i-lucide-component',
     to: '/pxcomponents',
   },
+  {
+    label: 'PxComponentsDefinitions',
+    icon: 'i-lucide-component',
```

Comment:
let's have different icons for each menu item. you can find one that is fitting here: https://lucide.dev/icons/